recipes/CommonVoice/ASR/transformer/extra_requirements.txt

```diff
@@ -0,0 +1,2 @@
+transformers
+datasets
```

Comment:
`datasets` is not needed and should not be a requirement imo

Reply:
I change the download dataset  function and remove the dependencies to dataset. Please check download_locale function is common_voice_prepare.py